lib/wallet/lightning_tx_detail.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart' hide Flow;
+import 'package:intl/intl.dart';
+import 'package:ten_ten_one/bridge_generated/bridge_definitions.dart';
+import 'package:ten_ten_one/utilities/tto_table.dart';
+
+class LightningTxDetail extends StatelessWidget {
+  static const subRouteName = 'ldk-tx';
+
+  final LightningTransaction transaction;
+
+  const LightningTxDetail({required this.transaction, super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final formatter = NumberFormat();
+    formatter.minimumFractionDigits = 0;
+    formatter.maximumFractionDigits = 0;
+    return Scaffold(
+      appBar: AppBar(title: const Text('Lightning Transaction Detail')),
```

Comment:
```suggestion
      appBar: AppBar(title: const Text('Lightning Payment Detail')),
```

I think transaction makes more sense in a Bitcoin context - I'd call it payment for Lightning. I think we did that for most parts of the application but am not sure it's consistent everywhere.